models/articles.models.js

```diff
@@ -47,3 +47,13 @@ exports.insertNewCommentOnArticle = (author, body, article_id) => {
       return result.rows[0];
     });
 };
+
+exports.updateArticleById = (inc_votes, article_id) => {
+  const queryStr = `UPDATE articles SET votes=votes+${inc_votes} WHERE article_id = $1 RETURNING *`;
```

Comment:
This is vulnerable to SQL injection.  To avoid this vulnerability, you should use parameterized queries for all user input, including inc_votes.  Please update.